api/src/main/resources/workbench.yaml

```diff
@@ -434,6 +442,7 @@ paths:
             $ref: "#/definitions/ResearchPurposeReviewRequest"
       responses:
         "200":
+          description: success
```

Comment:
Is description required? I ask because I judge this response to be self-describing (which may be why "success" is a perfectly reasonable description). To put it another way, does adding this add value?

Reply:
Unfortunately yes, it is required by the Swagger spec (removal fails validation). I agree this is already self-describing and that this description adds no value. It felt slightly better than empty string.

Reply:
Agreed and thanks for the info.